lib/Rfc1035StubResolver.php

```diff
@@ -191,7 +210,13 @@ public function reloadConfig(): Promise
         }
 
         $promise = call(function () {
-            $this->config = yield $this->configLoader->loadConfig();
+            try {
+                $this->config = yield $this->configLoader->loadConfig();
+                $this->configStatus = self::CONFIG_LOADED;
+            } catch (ConfigException $e) {
+                \trigger_error("Could not load DNS config, using synchronous fallback", E_USER_WARNING);
```

Comment:
I guess we have to add a `try` / `catch` here to handle throwing error handlers.

Reply:
Maybe: however, my application has a throwing error handler, but I've already added exceptions for all files in the amphp namespace, since other amphp libraries also trigger errors that surface to my handler.